pruner/src/statement_pruner.cpp

```diff
@@ -68,14 +68,17 @@ const IR::P4Program *prune_statements(const IR::P4Program *program,
                                       uint64_t prog_size) {
     int same_before_pruning = 0;
     int result;
-    int max_statements = prog_size / SIZE_BANK_RATIO;
+    // int max_statements = prog_size / SIZE_BANK_RATIO;
```

Comment:
Can you reenable this?

Reply:
Alright, but we might need to play around the ratio,